scripts/CEE/kubelet-restart/script.sh

```diff
@@ -24,33 +24,22 @@ finish_job(){
     echo "Job finished at $CURRENTDATE"
 }
 
-
-
 ## Function for kubelet restart
 restart_kubelet(){
-
     echo "Restarting Kubelet on \"${NODE}\"..."
-    cat <<EOF | oc -n default debug node/${NODE}
-    chroot /host 
-    systemctl restart kubelet
-EOF
-
-    if [ $? -eq 0 ]; then 
-        echo "[SUCCESS] Kubelet successfully restarted on \"${NODE}\" ."
+    if oc -n default debug node/"${NODE}" -- chroot /host systemctl restart kubelet; then
```

Comment:
Is this working as expected? The original commands are executed separated

Reply:
Yes Bo, it is working as expected.

Reply:
Here is the desired output
```
ocm backplane testjob logs openshift-job-dev-92gl2
Job started at 2024-08-19 07:29:15
..................................

Restarting Kubelet on "ip-10-0-0-172.ec2.internal"...
Starting pod/ip-10-0-0-172ec2internal-debug-p4hgp ...
To use host binaries, run `chroot /host`

Removing debug pod ...
[SUCCESS] Kubelet successfully restarted on "ip-10-0-0-172.ec2.internal".


..................................
Job finished at 2024-08-19 07:29:15
```